ERCS/erc-7586.md

```diff
@@ -0,0 +1,183 @@
+---
+eip: 7586
+title: Interest Rate Swaps
+description: Interest rate swaps derivative contracts
```

Comment:
```suggestion
description: On-chain implementation of derivative contracts swapping fixed and floating interest rates.
```

Feel free to continue improving this, but your description needs to be _more_ descriptive than just your title.